### PR TITLE
Ericalloyd caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @dumpsterfire

### PR DESCRIPTION
**Pull request by ericalloyd**
Edited the caption to reflect the image selected.

fixes #2

### CODE REVIEW

**Comment by allthedoll** [comment]
🦄

File: _posts/0000-01-04-selkins13.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Erica Lloyd loves organization!
```
Comment (ericalloyd) [inline _posts/0000-01-04-selkins13.md]: Do we need your last name here?

Comment (ericalloyd) [inline _posts/0000-01-04-selkins13.md]:
```suggestion
Erica loves organization!
```

**Review by ericalloyd — COMMENTED** [review]
Booo can't review your own PR

**Pull request merged**